src/Stripe.net/Services/_base/AnyOf.cs

```diff
@@ -103,14 +103,14 @@ public override Type Type
         /// </summary>
         /// <param name="value">The value to convert.</param>
         /// <returns>An <see cref="AnyOf{T1, T2}"/> object that holds the value.</returns>
-        public static implicit operator AnyOf<T1, T2>(T1 value) => new AnyOf<T1, T2>(value);
+        public static implicit operator AnyOf<T1, T2>(T1 value) => value == null ? null : new AnyOf<T1, T2>(value);
```

Comment:
Unfortunately, because this is a generic class, I don't think it's possible to write a helper method to do this in a single place.